docs/building-apps.md

```diff
@@ -218,6 +219,25 @@ my-app/
   webos-meta/
 ```
 
+## [contenthash] Injection
```

Comment:
This documentation describes not the `content-hash` option, but the inside code. Don't you think?